aospy/automate.py

```diff
@@ -384,10 +384,13 @@ def submit_mult_calcs(calc_suite_specs, exec_options=None):
             The region(s) over which any regional reductions will be performed.
             If 'all', use all regions in the ``regions`` attribute of each
             ``Proj``.
-        date_ranges : 'default' or tuple of datetime.datetime objects
+        date_ranges : 'default' or a list of tuples of datetime.datetime objects
             The range of dates (inclusive) over which to perform calculations.
             If 'default', use the ``default_start_date`` and
             ``default_end_date`` attribute of each ``Run``.
+            Else provide a list, each containing a tuple of start and end dates
```

Comment:
No reason for you to know this :), but in the master (i.e. unreleased) version of aospy we also support partial datetime strings (e.g. `'0001'`), `np.datetime64`, or `cftime.datetime` objects as well.  It would be great if you could update the docstring here to reflect that too.  Maybe something along the lines of:
```
Else provide a list of tuples, each containing a pair of start and end dates,
such as ``date_ranges=[(start, end)]`` where ``start`` and ``end`` are 
each ``datetime.datetime`` objects, partial datetime strings (e.g. '0001'),
``np.datetime64`` objects, or ``cftime.datetime`` objects.
```